config/permissions.go

```diff
@@ -0,0 +1,214 @@
+package config
+
+import "gopkg.in/macaroon-bakery.v2/bakery"
+
+var (
+	PricePermissions = []bakery.Op{
+		{
+			Entity: "market",
+			Action: "updatemarketprice",
+		},
+	}
+	MarketPermissions = []bakery.Op{
+		{
+			Entity: "market",
+			Action: "openmarket",
+		},
+		{
+			Entity: "market",
+			Action: "closemarket",
+		},
+		{
+			Entity: "market",
+			Action: "updatemarketstrategy",
+		},
+	}
+	ReadonlyPermissions = []bakery.Op{
+		{
+			Entity: "operator",
+			Action: "reportmarketfee",
+		},
+		{
+			Entity: "operator",
+			Action: "listswaps",
+		},
+		{
+			Entity: "operator",
+			Action: "listmarket",
+		},
+		{
+			Entity: "operator",
+			Action: "balancefeeaccount",
+		},
+	}
+
+	AdminPermissions = []bakery.Op{
```

Comment:
Admin is missing permissions for `claimfeedeposit` and `claimmarketdeposit`. Your branch needs to be synced with master.